stores/src/main/java/dataBase/DataBaseConnect.java

```diff
@@ -0,0 +1,25 @@
+package dataBase;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+public class DataBaseConnect {
+    private final String driver = "org.h2.Driver";
```

Comment:
if you create just one DbStorePopulator class, what for do you move this functionality to another class?